cob_bms_driver/src/fake_bms.py

```diff
@@ -74,9 +74,9 @@ def timer_cb(self, event):
 
     def timer_consume_power_cb(self, event):
         # emulate the battery usage based on the current values
-        self.remaining_capacity += (self.current/self.poll_frequency)/3600   
-        if self.remaining_capacity <= 0:
-            self.remaining_capacity = 0
+        self.remaining_capacity += round(((self.current/self.poll_frequency)/3600), 5)
```

Comment:
Won't this prevent the capacity from updating if the delta is very small and always rounded to 0? Is there a reason you need to round?